### PR TITLE
Add echo-service

### PR DESCRIPTION
**Pull request by figitaki**
Adds a simple Node service that connects to the GraphQL endpoint of a local node and will send back any payments it receives minus the associated transaction fee. See the `README` for more instructions.

**Remaining Tasks**
- [ ] Tests
- [ ] Improved logging

### CODE REVIEW

File: echo-service/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+This is a simple node service that listens for transactions to a specific address and then simply sends a payment back to the sender with the amount of `amount - fee`. The fee can be configured but by default is set to `5`.
+
+## Usage
+
+First you'll need to have a `coda` daemon running on your machine. See the docs [here](https://codaprotocol.com/docs/getting-started/) for instructions on getting a node, then run the following command:
+
+```
+$ coda daemon -rest-port 49370 -peer beta.o1test.net:8303
+```
```
Comment (bkase) [inline echo-service/README.md]: this also requires the daemon to track the wallet key right? (We need that for ownedWallets to work)

Comment (figitaki) [reply echo-service/README.md]: Yeah, I couldn't get it working with proving keys so I just created a wallet in the GraphiQL.

Comment (figitaki) [reply echo-service/README.md]: You can pass in a `PUBLIC_KEY` env var to set it manually otherwise it uses the first entry in the `ownedWallets` query.

Comment (Schmavery) [reply echo-service/README.md]: Our graphql docs should contain info about how to do the "import" workaround

**Pull request merged**